releases/release-5.3.4.md

```diff
@@ -0,0 +1,151 @@
+
++ PD
+
+    <!--owner: @nolouch-->
+
++ TiFlash
+
+    <!--compute owner: @zanmato1984-->
+
+    <!--storage owner: @flowbehappy-->
+
++ Tools
+
+    + Backup & Restore (BR)
+
+    <!--owner: @3pointer-->
+
+    + Dumpling
+
+    <!--owner: @niubell-->
+
+    + TiCDC
+
+    <!--owner: @nongfushanquan-->
+
+    + TiDB Binlog
+
+    <!--owner: @niubell-->
+
+    + TiDB Lightning
+
+    <!--owner: @niubell-->
+
+    + TiDB Data Migration (DM)
+
+    <!--owner: @niubell-->
+
+## Bug fixes
+
++ TiDB
+
+    <!--sql-infra owner: @Defined2014-->
+
+    - (dup) Fix the issue that the Region cache is not cleaned up in time when the Region is merged[#37141](https://github.com/pingcap/tidb/issues/37141)
+    - (dup) Fix the issue that TiDB writes wrong data due to the wrong encoding of the ENUM or SET column [#32302](https://github.com/pingcap/tidb/issues/32302)
+    - (dup) Fix the issue that database-level privileges are incorrectly cleaned up [#38363](https://github.com/pingcap/tidb/issues/38363)
+    - (dup) Fix the issue that the `grantor` field is missing in the `mysql.tables_priv` table [#38293](https://github.com/pingcap/tidb/issues/38293)
+    - (dup) Fix the issue that `KILL TIDB` cannot take effect immediately on idle connections [#24031](https://github.com/pingcap/tidb/issues/24031)
+    - change date_add and date_sub string_(int/string/real/decimal) function return type to string [#36394](https://github.com/pingcap/tidb/issues/36394)
+    - fix(parser): restore table option INSERT_METHOD should use WriteKeyWord [#38368](https://github.com/pingcap/tidb/issues/38368)
+    - fix authentication with MySQL 5.1 and older clients [#29725](https://github.com/pingcap/tidb/issues/29725)
+
+    <!--executor owner: @zanmato1984-->
+
+    - (dup) Fix wrong results of `GREATEST` and `LEAST` when passing in unsigned `BIGINT` arguments [#30101](https://github.com/pingcap/tidb/issues/30101)
+    - (dup) Fix the issue that the result of `concat(ifnull(time(3))` in TiDB is different from that in MySQL [#29498](https://github.com/pingcap/tidb/issues/29498)
+    - avoid sum from avg overflow [#29952](https://github.com/pingcap/tidb/issues/29952)
+    - add an unit test case for unreasonable invoking Close [#30587](https://github.com/pingcap/tidb/issues/27125)
+    - HashJoinExec checks the buildError even if the probeSide is empty [#30289](https://github.com/pingcap/tidb/issues/30289)
+    - expression: resize the result for IfXXSig [#37414](https://github.com/pingcap/tidb/issues/37414)
+    - change date_add and date_sub string_(int/string/real/decimal) function return type to string. [#27573](https://github.com/pingcap/tidb/issues/27573)
+    - fix hashjoin goleak [#39026](https://github.com/pingcap/tidb/issues/39026)
+    - fix: the results of tikv and tiflash are different [#37258](https://github.com/pingcap/tidb/issues/37258)
+
+    <!--transaction owner: @cfzjywxk-->
```

Comment:
- Fix the issue explain analyze with DML executors may respond to the client before the transaction commit has finished. [#37373](https://github.com/pingcap/tidb/issues/37373)

Reply:
Added